.circleci/scripts/trigger-dockerhub.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#
+# Trigger a dockerhub build.
+#
+
+for ENDPOINT in "${!DOCKERHUB_TRIGGER_@}"; do
```

Comment:
For anyone else watching, this is looping through these variables
<img width="354" alt="Screen Shot 2020-07-16 at 1 40 55 pm" src="https://user-images.githubusercontent.com/121107/87624040-115d2800-c76a-11ea-8d91-43fb44a1eef8.png">